scripts/test/sample_transmission_calculator/stc_view_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+import sys
+from unittest.mock import Mock, patch
+from mantidqt.utils.qt.testing import start_qapplication
+from SampleTransmissionCalculator.stc_view import SampleTransmissionCalculatorView
+from qtpy.QtWidgets import QHBoxLayout
+
+
+@start_qapplication
+class SampleTransmissionCalculatorViewTest(unittest.TestCase):
+    def setUp(self):
+        self.view = SampleTransmissionCalculatorView()
+
+    @unittest.skipIf(not hasattr(QHBoxLayout, 'replaceWidget'), 'Invalid qt version')
+    def test_get_input_dict(self):
+        self.view.binning_type_combo_box.setCurrentIndex(0)
+        self.view.single_low_spin_box.setValue(0.1)
+        self.view.single_width_spin_box.setValue(0.1)
+        self.view.single_high_spin_box.setValue(10.0)
+        self.view.multiple_line_edit.setText('0.1,0.1,20.0')
+        self.view.chemical_formula_line_edit.setText('C')
+        self.view.density_combo_box.setCurrentIndex(1)
+        self.view.density_spin_box.setValue(0.2)
+        self.view.thickness_spin_box.setValue(0.2)
+
+        input_dict = self.view.get_input_dict()
+
+        input_dict_expected = {
+            'binning_type': 0,
+            'single_low': 0.1,
+            'single_width': 0.1,
+            'single_high': 10.0,
+            'multiple_bin': '0.1,0.1,20.0',
+            'chemical_formula': 'C',
+            'density_type': 'Number Density',
+            'density': 0.2,
+            'thickness': 0.2,
+        }
+        self.assertEqual(input_dict, input_dict_expected)
+
+    @unittest.skipIf(not hasattr(QHBoxLayout, 'replaceWidget'), 'Invalid qt version')
```

Comment:
Given these are running in QT5 now we should be able to remove these `unittest.skipIf` statements now.